vispy/visuals/box.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Vispy Development Team. All Rights Reserved.
```

Comment:
I think I've changed all of these to 2015s in the rest of the code base now, please change in any files you've added in this PR